spec/acceptance/indexConsistentTest.spec.ts

```diff
@@ -0,0 +1,70 @@
+import {assert as assert} from 'chai';
+import { DBTable, DBColumn } from '../../src';
+import { NodeNamo } from '../../src/nodeNamo';
+import Config from './config';
+import { DocumentClient } from 'aws-sdk/clients/dynamodb';
+import { Const } from '../../src/const';
+
+@DBTable({name:'nodenamo_acceptance_indexConsistentTest'})
```

Comment:
Could you add stronglyConsistent tests to idTes, hasTest, and hashRangetest?  For each file, make sure you added test for get(), list(), and find()?

Reply:
Added requested tests to next commit